content/_index.md

```diff
@@ -0,0 +1,23 @@
+# Linux as Firmware
```

Comment:
I'm not sure I would want this as the index page. This feels like it has too much detail about what's happening and not enough "what the hell is this?".

I feel like this belongs on a Details page.